addons/point_of_sale/tests/test_pos_basic_config.py

```diff
@@ -926,5 +926,10 @@ def test_invoice_past_order(self):
             order_to_invoice.write({'partner_id': test_customer.id})
             order_to_invoice.action_pos_order_invoice()
             # check invoice
-            self.assertTrue(order_to_invoice.account_move, 'Invoice should be created.')
-            self.assertTrue(order_to_invoice.account_move.invoice_date != order_to_invoice.date_order.date(), 'Invoice date should not be the same as order date since the session was closed.')
+            invoice = order_to_invoice.account_move
+            self.assertTrue(invoice, 'Invoice should be created.')
+            self.assertNotEqual(invoice.invoice_date, order_to_invoice.date_order.date(), 'Invoice date should not be the same as order date since the session was closed.')
+
+            # check that the payment date is set to the payment date and not to the invoice_date
```

Comment:
```suggestion
            # check that the payment date is set to the order date and not to the invoice_date
```

Reply:
In any case it's tested and should be fine, please correct the comment and we can merge it